### DISCUSSION TITLE
Anyone try Laravel 9  in Gitpod + DDEV

### DISCUSSION THREAD

**Discussion by wahyusa** [ddev/ddev#4450] [Q&A]
I am just start using DDEV in gitpod.

Everything works fine following this official docs 
https://ddev.readthedocs.io/en/stable/users/quickstart/#laravel

The only problem is the assets URL was not point in to the right gitpod URL, it just point back to `laravel.ddev.test:5173` rather than my gitpod subdomain.

It also still using this default style port url `:5173` which in gitpod it should be inside the url like `5173-gitpod-domain.gitpod.io`

Have try to modify the VITE_URL in `.ddev/config.yaml`

```yaml
web_environment:
    - VITE_APP_URL=8080-gitpod.region.gitpod.io
```

or

```yaml
web_environment:
    - VITE_APP_URL=5173-gitpod.region.gitpod.io
```

or even remove the PORT env in `.ddev/.env`
to something like
```env
VITE_PRIMARY_PORT=
VITE_SECONDARY_PORT=
```

still doens't work and I don't even found any opened port containing 5173 in gitpod itself.

Hope it will fixed soon or anyone have same problem too ?

**Comment by tyler36** [comment]
Hi @wahyusa  

If you are using [ddev-viteserve](https://github.com/torenware/ddev-viteserve), it might be best to open an issue over there. 

@torenware has spent a lot of time working with DDEV and vite and will probably have an insight or 2.
